src/database/models/reportQueries.js

```diff
@@ -12,13 +12,16 @@ module.exports = (sequelize, DataTypes) => {
 			},
 			report_code: {
 				type: DataTypes.STRING,
+				primaryKey: true,
```

Comment:
@sumanvpacewisdom remove id as primary key, keep only report_code as primary key

Reply:
@sumanvpacewisdom add indexes for report_code